lisp/ess-inf.el

```diff
@@ -3073,6 +3074,20 @@ subprocess and Emacs buffer `default-directory'."
       (error "Not implemented for dialect %s" ess-dialect))))
 
 (defalias 'ess-change-directory 'ess-set-working-directory)
+(define-obsolete-function-alias
+  'ess-use-dir 'ess-set-working-directory "2018-06-11")
+
+(defun ess-use-this-dir (&optional no-force-current)
+  "Set the current process directory to the directory of this file.
+
```

Comment:
Small nitpick: there shouldn't be a newline after the leading sentence.

Reply:
Removed